front/app/api/areas/useAreas.ts

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import eventsKeys from './keys';
+import { IAreas, AreasKeys, IAreasQueryParams } from './types';
+
+const fetchEvents = (filters: IAreasQueryParams) => {
+  const {
+    pageNumber,
+    pageSize,
+    forHomepageFilter: for_homepage_filter,
+    includeStaticPages,
+  } = filters;
+  return fetcher<IAreas>({
+    path: '/areas',
+    action: 'get',
+    queryParams: {
+      'page[number]': pageNumber || 1,
+      'page[size]': pageSize || 5000,
```

Comment:
I don't think we'd want to query 5k areas at once. Unlikely to ever happen, but 100 is not unrealistic. Not sure how fast/slow that would be.

Reply:
This is how it was implemented before - it is unlikely to have 5000 areas but I guess the amount was set to a high number to make sure it covers all cases. In any case, the backend will return as many as there a re so I don't think this will impact performance in any way. If we see that it does in the future, we can always think of implementing pagination in some form.